models/tickets.py

```diff
@@ -80,7 +80,7 @@ def find_count_by_status(cls, status):
     def find_count_by_age_status(cls, status, minutes):
         #calculated in minutes: 1 day = 1440, 1 week = 10080
         dateTime = datetime.now() - timedelta(minutes = minutes)
-        return db.session.query(TicketModel).filter(TicketModel.updated >= dateTime).filter(TicketModel.status == status).count()
+        return db.session.query(TicketModel).filter(TicketModel.updated_at >= dateTime).filter(TicketModel.status == status).count()
```

Comment:
Do you know what this method is for? I'm curious how we are using it.

Reply:
Looks like find_count_by_age_status is used for the dashboard widgets

Reply:
OK. I will ask Hugh about this the next time we meet. Looking at this method I don't think it works as intended, and there are no tests to refer to.